src/main/java/pl/edu/mimuw/logic/formulas/duble/andFormula.java

```diff
@@ -0,0 +1,15 @@
+package pl.edu.mimuw.logic.formulas.duble;
+
+import pl.edu.mimuw.logic.Formula;
+import pl.edu.mimuw.logic.Valuation;
+
+public class andFormula extends doubleFormulaA {
```

Comment:
Trzymajmy konwencji z nazwenictwem klas z dużych liter, bo wszyscy tak robią, to nie jest coś co zależy od tego jak nam się podoba, a rozróznia kalsy od funkcji